packages/sync/src/orbitStoreSyncInterface.ts

```diff
@@ -0,0 +1,59 @@
+import { OrbitStore } from "@withorbit/store-shared";
+import {
+  AttachmentID,
+  AttachmentMIMEType,
+  Event,
+  EventID,
+} from "@withorbit/core2";
+import { SyncInterface } from "./syncInterface";
+
+export class OrbitStoreSyncInterface implements SyncInterface {
```

Comment:
nit: `OrbitStoreSyncManager` instead maybe? Wouldn't `Interface` in the name imply an abstract implementation?

Reply:
Good point—in TypeScript, I suppose it would. Perils of working in so many languages…

This is really an "adapter" in the Gang of Four sense: the idea is that you can sync from/to several kinds of things (the server via an `APIClient`, another local `OrbitStore`, etc). I wouldn't want to call it a "manager" because it doesn't do anything itself; it just translates requests to something else which does the work.